### PR TITLE
Npande tf 23aug

### PR DESCRIPTION
**Pull request by npande**

### CODE REVIEW

File: aws-observability-terraform/README.md [hunk 1]
```diff
@@ -1,79 +1,41 @@
...
-  aws_account_alias         = "production"
-  sumologic_organization_id = "0000000000123456"
-}
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13.0 |
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 3.42.0 |
+| <a name="requirement_random"></a> [random](#requirement\_random) | >= 3.1.0 |
+| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | >= 2.9.0 |
+| <a name="requirement_time"></a> [time](#requirement\_time) | >= 0.7.1 |
```
Comment (arunpatyal) [inline aws-observability-terraform/README.md:10]: Isn't REST API provider also needed by various modules?

File: aws-observability-terraform/app-modules/README.md [hunk 2]
```diff
@@ -3,14 +3,15 @@
 | Name | Version |
 |------|---------|
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13.0 |
-| <a name="requirement_null"></a> [null](#requirement\_null) | ~> 2.1 |
-| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | ~> 2.6.2 |
+| <a name="requirement_null"></a> [null](#requirement\_null) | >= 2.1 |
+| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | >= 2.6.2 |
```
Comment (arunpatyal) [inline aws-observability-terraform/app-modules/README.md:8]: rest api provider?

**Review by arunpatyal — CHANGES_REQUESTED** [review]

File: aws-observability-terraform/README.md (hunk 1 above)
Comment (npande) [reply aws-observability-terraform/README.md:10]: Not using restAPI provider anywhere in AWS Obs Soln.

File: aws-observability-terraform/app-modules/README.md (hunk 2 above)
Comment (npande) [reply aws-observability-terraform/app-modules/README.md:8]: Not using restAPI provider anywhere in AWS Obs Soln.

**Comment by arunpatyal** [comment]
LGTM

**Pull request merged**